history/warren-weaver.md

```diff
@@ -0,0 +1,11 @@
+---
+description: One of the pioneers of machine translation
+---
+# Warren Weaver
+Warren Weaver was an American scientist who designed several methods for developing Machine translation in 1949.
+
+The methods included the use of wartime cryptography techniques, Shannon’s information theory, 
+statistical analysis and juxtaposition of logic and universal features of language.
+
+With this method, research in Machine translation was ignited and the first conference was convened. 
+The submitted papers leaned on controlled languages, construction of a sublanguage, as well as pre- and post-editing. 
```

Comment:
The first conference probably deserves a separate article of its own.  Since these were not his papers, right?

Reply:
No, those were not his papers.